aksel.nav.no/package.json

```diff
@@ -8,8 +8,7 @@
     "dev": "yarn workspace website dev",
     "build:next": "yarn workspace website build",
     "serve:next": "yarn workspace website start",
-    "sanity:update": "yarn workspace website update:sanity",
-    "test": "yarn workspace website test"
```

Comment:
kjører vi ikke tester lenger for aksel.nav.no?

Reply:
Flyttet den hit: https://github.com/navikt/aksel/pull/2723/files#diff-9a56ad11d152da97d7a301bf5761dd7379b7a4b73aa3fa158b195de048d882eeR27